src/Autowhatever.js

```diff
@@ -235,4 +241,28 @@ export default class Autowhatever extends Component {
       </div>
     );
   }
+
+  componentDidUpdate() {
+    if (this.refs.focusedItem) {
+      const itemNode = findDOMNode(this.refs.focusedItem);
+      const containerNode = findDOMNode(this.refs.itemsContainer);
```

Comment:
Per [ReactDOM.findDOMNode](https://facebook.github.io/react/docs/top-level-api.html#reactdom.finddomnode), you don't need to use the function if a ref is attached.  So yeah, `const { focusedItem, itemsContainer } = this.refs;` should work.